make-for-each.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#set -x
+#echo "invokation: $@"
+#echo "command: ${@:2}"
+cmd=${@:2}
+c=0
+declare -a fail
+for d in $(ls -d */); do
+  cd $d
+  if ! [ -e Makefile ]; then
+    echo && echo SKIPPING TESTS in $d because there is no Makefile
+  else
+    echo && echo "Making $1 in $d" && echo
+    make $1
+    if [ $? != 0 ]; then
+      fail[$c]=$d
+      c=$((c+1))
+      echo && echo "$1 in $d FAILED" && echo
+      if [ $1 == test-ci ]; then 
+        echo && echo "LocalStack logs for $d" && echo
+        cat logs.txt
+      fi
+    fi
+    # TODO: do we still need an extra command, and if yes, should it be executed regargless of the make success?
```

Comment:
Seems to be a leftover here as well?

Reply:
I guess; it was left on purpose, however nothing really is left to utilize this, so might as remove the command. The target that made use of it turned out to be conceptually flawed.